src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160613_workflow_execution_url.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="davidan" id="workflow_execution_instance">
+        <addColumn tableName="WORKFLOW">
+            <column name="EXEC_SERVICE_KEY" type="VARCHAR(32)">
+            </column>
+        </addColumn>
+
+        <createIndex indexName="idx_workflow_exec_service_key" tableName="WORKFLOW" unique="false">
+            <column name="EXEC_SERVICE_KEY"/>
+        </createIndex>
```

Comment:
I debated on including an index here. We currently don't query workflows by their cromwell instance, but I suspect we'll be doing that soon. Is this premature optimization?